### ISSUE TITLE
Variable `org-roam-node-backlinkscount-number` isn't defined

### ISSUE THREAD

**Issue by maikol-solis** [Vidianos-Giannitsis/zetteldesk.el#4]
Hi! 

I tried to run `zetteldesk-node-insert-sort-backlinks` over a node and I got this error

`org-roam-node-sort-by-backlinks: Symbol’s function definition is void: org-roam-node-backlinkscount-number`

Thanks!

**Comment by Vidianos-Giannitsis** [comment]
Shit, I replied via email 1 and a half days ago and just know I see that it was never sent. Well, here is the original reply

Oh wow, I should have noticed this one earlier. So I wrote this function and the sorting function but I also made a cl-defmethod which counts the amount of backlinks the node has and returns that number. But as that is a generic function, I added that to my main emacs config, but was supposed to also copy it over to the dependencies section of the package.

Somehow I missed that, so this problem has been here from the start but it was hard for me to notice as I loaded that function at the same time as the package. If you pull down the latest commit, it should be working now.
